AutoResizeFont/DynamicLabel.swift

```diff
@@ -10,38 +10,68 @@ import UIKit
 
 @IBDesignable
 final class DynamicLabel: UILabel {
+    
+    
+    
     @IBInspectable
-    var fontSizePts: CGFloat {
-        didSet {
+    var fontSize: CGFloat {
+        get{
```

Comment:
Why not using `didSet`?

Reply:
didSet does not give me direct access to newValue on update. if i am to use the value immediately it is set, I need to use set.

Reply:
Or you can use `willSet`